test/mpi/maint/jenkins/xfail.conf

```diff
@@ -91,3 +91,7 @@
 mpich-main-arm.* * * * * /^sendflood /          xfail=ticket0       pt2pt/testlist
 mpich-main-arm.* * * * * /^nonblocking3 /       xfail=ticket0       coll/testlist
 mpich-main-arm.* * * * * /^alltoall /           xfail=ticket0       threads/pt2pt/testlist
+
+# pmix doesn't work well with ch3 under oversubscription, ref. PR5984
```

Comment:
What is the result of the oversubscription policy? I'm guessing you mean the policy of the Open MPI launcher (orte)? We had logic in the runtests script to allow oversubscription when PMIx testing was setup.

Reply:
My mistake, the logic is in the Jenkins scripts repo:
```
MPIEXEC="$HOME/software/openmpi-3.0.0/bin/mpiexec -oversubscribe"
```

Reply:
It launches the oversubscribed processes and it works for ch4+pmix. But ch3 uses lazy connection so it will look for the businesscard from PMI upon the first communication. Somehow PMIX will return key not found for some of the process even though the key was put. I don't know what is the real cause yet. Apparently it only occurs in oversubscribed tests and only ch3 and only with pmix.

Reply:
Ah, OK. That is a different issue. We are probably overdue to upgrade the tools used to test the PMIx integration. That can be some another time, though.